prboom2/src/gl_missingtexture.c

```diff
@@ -61,12 +61,22 @@ typedef struct
   sector_t **list;  // list of adjoining sectors
 } fakegroup_t;
 
+typedef struct
+{
+    sector_t *source; // The sector to receive a fake bleed-through flat
+    sector_t *target; // The floor sector whose properties should be copied
+    int ceiling;      // Ceiling or floor
+} bleedthrough_t;
+
 static int numfakeplanes = 0;
 static fakegroup_t *fakeplanes = NULL;
 static sector_t **sectors2 = NULL;
+static bleedthrough_t *bleedsectors = NULL;
+static int numbleedsectors = 0;
```

Comment:
Where is this ever reset? What happens when entering the next map?

Reply:
Good catch, I needed the reset at the top of that loop. Fixed.